.github/CHANGELOG.md

```diff
@@ -26,108 +26,109 @@
+62. [PFD] Added option to sync FD/LS buttons between CPT and FO - @Eagle941 (Joe)
+63. [EFB] Add automatic loading of a lighting preset - @SpiritZephyr (Ben)
+64. [COND] Connect Air Conditioning System to Electrical System - @mjuhe (Miquel Juhe)
+65. [HYD] Default discrete of failed ADIRs impacts gear safety valve correctly - @Crocket63 (crocket)
+66. [ADDON] Simbridge Integration - @lucky38i (Alex)
+67. [SD] Visual and functional improvements/fixes to the HYD SD page - @lukecologne (luke)
+68. [HYD] Added optional auxiliary hydraulic section in core hydraulic circuits - @Crocket63
+69. [EFB] Added SimBridge Health Check icon to Status Bar - @frankkopp (Frank Kopp)
+70. [HYD] Fixed opened gear door wobbling - @Crocket63
+71. [HYD] Fix gear extending whith dual lgciu power loss - @Crocket63 (crocket)
+72. [EFB] flyPad pushback option to ignore controller inputs - @frankkopp (Frank Kopp)
+73. [FCTL] Add realistically simulated F/CTL computers: ELAC/SEC/FCDC/FAC - @lukecologne (luke)
+74. [HYD] Accelerations impacts hydraulic reservoir fluid movement - @Crocket63 (crocket)
+75. [FLIGHTMODEL] Add SU10 crosswind parameters - @donstim (donbikes#4084)
+76. [MODEL] Visible pilot avatars via flyPad setting - @frankkopp (Frank Kopp)
+77. [MODEL] Fixed screens flickering in SU10 - @tyler58546 (tyler58546)
+78. [EFB] Added Local Files support - @ErickSharp (Erick Torres) @frankkopp (Frank Kopp)
+79. [BLEED] Added custom Wing Anti-Ice model - @omrygin, @Eagle941 (Joe)
+80. [MISC] Fix FDR data for support analysis - @frankkopp (Frank Kopp)
+81. [LIGHTS] Increased max brightness of cockpit displays - @frankkopp (Frank Kopp)
+82. [EFB] Improved Aircraft Presets procedures - @frankkopp (Frank Kopp)
+83. [FMGC] Make DES SPD LIM available prior to descent - @tracernz (Mike)
+84. [LIGHTS] Redone cockpit emissives panels and ambient lighting -@FinalLightNL (FinalLight)
+85. [FMGC] Rework f-pln departure page - @tracernz (Mike)
+86. [MCDU] Support coroute loading and viewing - @lucky38i (Alex)
+87. [HYD] Fixed autobrake controller tuning - @Crocket63 (crocket)
+88. [ENGINE] Improved engine primary parameters performance (EGT & Fuel Flow) - @Taz5150 (TazX [Z+2]#0405)
+89. [ENGINE] Fix Engine Flex Calculations - @Taz5150 (TazX [Z+2]#0405)
+90. [FLIGHTMODEL] Flight Model update - @donstim (donbikes#4084)
+91. [HYD] Fix incorrect gain in down pitch commands - @Crocket63 (crocket)
+92. [PFD] Improve appearance of L/DEV and V/DEV scales - @tracernz (Mike)
+93. [HYD] Implemented Electro Hydrostatic actuators - @Crocket63 (crocket)
+94. [MISC] Change 2 way overhead switches to be also controllable with mouse wheel up/down @patsta32 (Pat)
+95. [FAC] Move Speedscale computation to FAC - @lukecologne (luke)
+96. [EFB] Added deboarding button to flyPad Payload - @frankkopp (Frank Kopp)
+97. [EFB] Improved simbridge-client connection handling - @frankkopp (Frank Kopp)
+98. [HYD] Tilting gear mechanism - @Crocket63 (crocket)
```

Comment:
Please do not change this, 1. works for bulleting. 
```suggestion
2. [MCDU] Allow Wind Request from Simbrief flight plan - @USA-RedDragon (Jacob McSwain)
3. [HYD] Fix fluid return handling of actuators - @Crocket63 (crocket)
4. [ENGINE] Fuel persistency between each flight - @juliansebline (Julian Sebline#8476)
5. [HYD] Now allowing reverting gravity gear extension - @Crocket63 (crocket)
6. [TCAS] Fixed issue when turning to STBY while RA is issued - @2hwk (2Cas#1022)
7. [GPWS] Alt callouts now finish before altitude is reached - @2hwk (2Cas#1022)
8. [ENGINE] Improved engine model at TOGA - @Taz5150 (TazX [Z+2]#0405)
9. [FLIGHTMODEL] Improve pitch trim and manual braking level - @donstim (donbikes#4084)
10. [ATSU] Fix cargo weight is not set on manually PAX - @Revyn112 (Revyn112#1010)
11. [CDSS] When off no longer displays door video - @davidwalschots (David Walschots)
12. [FMGC] Map leg course with station declination where required - @tracernz (Mike)
13. [HYD] Gear system now supports spawning in flight gear up - @Crocket63 (crocket)
14. [MCDU] Added imperial runway length - @derl30n (Leon)
15. [MCDU] Refactor and improve input handling and conversions in MCDU W&B - @sidnov (Sid)
16. [MCDU] Improved arrivals/departure page scrolling, more true-to-life behaviour and cosmetic apperance - @2hwk ( 2Cas#1022)
17. [PFD] Show yellow GS reference line in correct conditions - @saschl (saschl#9432)
18. [ECAM] Fix erroneous SLATS NOT IN T.O CONFIG warning during flaps 3 takeoff - @beheh (Benedict Etzel)
19. [MODEL] Improved rivet mesh with more variation and detail - @Grinde (Grinde#4017)
20. [ELEC] Make battery voltmeters update one digit at a time - @beheh (Benedict Etzel)
21. [EWD] E/WD visual improvements - @lukecologne (luke)
22. [SFCC] Add SFCC bus outputs - @lukecologne (luke)
23. [EWD] Use FPPU angles for flaps/slats display - @lukecologne (luke)
24. [FMGC] Basic RNP at or below 0.3 support - @tracernz (Mike)
25. [ND] Add TERR ON ND peaks mode map visualization - @Sven [de en] (Sven Czarnian)
26. [SD] Improve F/CTL and WHEEL SD pages visuals - @lukecologne (luke)
27. [MCDU] Added formatter to improve text alignment and ease integration - @derl30n (Leon)
28. [EFB] Fix default unit to match UI and other consumers - @tracernz (Mike)
29. [HYD] Added possibility to lock or limit hydraulic actuators velocity - @Crocket63 (crocket)
30. [HYD] Added a new THS mechanism simulation - @Crocket63 (crocket)
31. [HYD] More prox sensors and gear actuator failures - @Crocket63 (crocket)
32. [MCDU] Added 4:3 aspect ratio compatibility to remote mcdu client - @tyler58546 (tyler58546)
33. [HYD] Fixed too slow leak measurement valves operation - @Crocket63 (crocket)
34. [ISIS] Added temporary ISIS font with arrows - @aweissoertel (Alexibexi#7550)
35. [PFD] Improve PFD barberpole rendering and behaviour - @lukecologne (luke)
36. [HYD] Fixed actuator position control demand consistency - @Crocket63 (crocket)
37. [FMGC] Only emit decel point when an approach is selected - @tracernz (Mike)
38. [HYD] Fixed SFCC computer failing to send FPPU commands - @Crocket63 (crocket)
39. [MCDU] Fix padding of arc radii on F-PLN - @tracernz (Mike)
40. [FMGC] Allow stringing of STARs with non-runway approaches - @tracernz (Mike)
41. [SD] Improve COND page visuals, fix some visual bugs on the SD - @lukecologne (luke)
42. [HYD] Aerodynamic forces applied on gear system - @Crocket63 (crocket)
43. [HYD] Fixed Lgciu state machine when reverting gravity extension - @Crocket63 (crocket)
44. [FMGC] Fix unnatural turn direction for path capture transition - @tracernz (Mike)
45. [FMGC] Fix unnatural turn direction for course capture transition - @tracernz (Mike)
46. [FMGC] Update spoiler CD for MSFS SDK clarification - @donstim (donbikes#4084)
47. [HYD] Actuators damping chambers can be asymetrical - @Crocket63 (crocket)
48. [FMGC] Implement procedure turns (PI leg) - @tracernz (Mike)
49. [ECAM] Move EWD to correct AC bus - @tracernz (Mike)
50. [FMGC] Fix inbound leg time for holds - @tracernz (Mike)
51. [MCDU] Improved visuals of Init-A and Init-B page - @derl30n (Leon)
52. [MODEL] Added new animated gear gravity extension handle- @tyler58546 (tyler58546), @MoreRightRudder (Mike), @Crocket63 (crocket), @Lantarius
53. [HYD] Custom Lvar for gear lever to fix ground collision bug - @Crocket63 (crocket)
54. [HYD] Randomised per actuator flow restrictions at plane init - @Crocket63 (crocket)
55. [MCDU] Hide stored elements on A/C Status when there are none - @tracernz (Mike)
56. [FMGC] Fix ident for CD legs - @tracernz (Mike)
57. [HYD] Refactor plane accelerations and added filtering to handle ground collisions - @Crocket63 (crocket)
58. [FMGC] Improve procedure loading performance - @tracernz (Mike)
59. [EFB] Added Payload Loading to EFB - @2hwk (2Cas#1022)
60. [MODEL] Remove blue highlights in lock mode - @tracernz (Mike)
61. [RMP] Fixed the behavior of the RMPs and ACPs (VHFs only) - Julian Sebline (Julian Sebline#8476 on Discord)
62. [PFD] Added option to sync FD/LS buttons between CPT and FO - @Eagle941 (Joe)
63. [EFB] Add automatic loading of a lighting preset - @SpiritZephyr (Ben)
64. [COND] Connect Air Conditioning System to Electrical System - @mjuhe (Miquel Juhe)
65. [HYD] Default discrete of failed ADIRs impacts gear safety valve correctly - @Crocket63 (crocket)
66. [ADDON] Simbridge Integration - @lucky38i (Alex)
67. [SD] Visual and functional improvements/fixes to the HYD SD page - @lukecologne (luke)
68. [HYD] Added optional auxiliary hydraulic section in core hydraulic circuits - @Crocket63
69. [EFB] Added SimBridge Health Check icon to Status Bar - @frankkopp (Frank Kopp)
70. [HYD] Fixed opened gear door wobbling - @Crocket63
71. [HYD] Fix gear extending whith dual lgciu power loss - @Crocket63 (crocket)
72. [EFB] flyPad pushback option to ignore controller inputs - @frankkopp (Frank Kopp)
73. [FCTL] Add realistically simulated F/CTL computers: ELAC/SEC/FCDC/FAC - @lukecologne (luke)
74. [HYD] Accelerations impacts hydraulic reservoir fluid movement - @Crocket63 (crocket)
75. [FLIGHTMODEL] Add SU10 crosswind parameters - @donstim (donbikes#4084)
76. [MODEL] Visible pilot avatars via flyPad setting - @frankkopp (Frank Kopp)
77. [MODEL] Fixed screens flickering in SU10 - @tyler58546 (tyler58546)
78. [EFB] Added Local Files support - @ErickSharp (Erick Torres) @frankkopp (Frank Kopp)
79. [BLEED] Added custom Wing Anti-Ice model - @omrygin, @Eagle941 (Joe)
80. [MISC] Fix FDR data for support analysis - @frankkopp (Frank Kopp)
81. [LIGHTS] Increased max brightness of cockpit displays - @frankkopp (Frank Kopp)
82. [EFB] Improved Aircraft Presets procedures - @frankkopp (Frank Kopp)
83. [FMGC] Make DES SPD LIM available prior to descent - @tracernz (Mike)
84. [LIGHTS] Redone cockpit emissives panels and ambient lighting -@FinalLightNL (FinalLight)
85. [FMGC] Rework f-pln departure page - @tracernz (Mike)
86. [MCDU] Support coroute loading and viewing - @lucky38i (Alex)
87. [HYD] Fixed autobrake controller tuning - @Crocket63 (crocket)
88. [ENGINE] Improved engine primary parameters performance (EGT & Fuel Flow) - @Taz5150 (TazX [Z+2]#0405)
89. [ENGINE] Fix Engine Flex Calculations - @Taz5150 (TazX [Z+2]#0405)
90. [FLIGHTMODEL] Flight Model update - @donstim (donbikes#4084)
91. [HYD] Fix incorrect gain in down pitch commands - @Crocket63 (crocket)
92. [PFD] Improve appearance of L/DEV and V/DEV scales - @tracernz (Mike)
93. [HYD] Implemented Electro Hydrostatic actuators - @Crocket63 (crocket)
94. [MISC] Change 2 way overhead switches to be also controllable with mouse wheel up/down @patsta32 (Pat)
95. [FAC] Move Speedscale computation to FAC - @lukecologne (luke)
96. [EFB] Added deboarding button to flyPad Payload - @frankkopp (Frank Kopp)
97. [EFB] Improved simbridge-client connection handling - @frankkopp (Frank Kopp)
98. [HYD] Tilting gear mechanism - @Crocket63 (crocket)
```
Please undo this change, 1. works for the changelog bulleting.